### PR TITLE
Add Puppet contributors to CLA signers

### PR DESCRIPTION
**Pull request by impl**
This change adds myself and two Puppet employees, @GabrielNagy and @ciprianbadescu, to the approved CLA signers list in support of #960.

## Proposed Commit Message

```
tools: Add Puppet contributors to CLA signers
```

### CODE REVIEW

**Review by TheRealFalcon — APPROVED** [review]

**Comment by TheRealFalcon** [comment]
Hi @impl , thanks for the PR. This branch needs to be rebased before it can be merged. Usually us maintainers do that before performing the merge, but I think you've disallowed that functionality. On the side panel of the PR, there should be a checkbox that says something like `Allow edits from maintainers.` In the future, if you make sure that is checked, then we can do the rebase automatically.

**Comment by GabrielNagy** [comment]
Hey @TheRealFalcon, I think I was able to rebase since the fork was on the puppetlabs org.

**Pull request merged**
